### PR TITLE
feat: Added Bennett.md

### PR DESCRIPTION
**Pull request by bsmrdel101**
- Added new branch: feat--2022-bennett
- Created Bennett.md
- Filled out required information inside the file

#### To Validate
  1. Make sure all GitHub action PR checks have passed.
  2. Pull down all related branches.

### CODE REVIEW

File: about-the-apprentices/2022/Bennett.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Hello there
+
+Hello everyone! My name is `Bennett` and I'm a new apprentice at Sparkbox! Right now, I'm working remotely from `Minnesota`, which means I'm on `Central` time and my preferred pronouns are `He/Him`.
```
Comment (marissahuysentruyt) [inline about-the-apprentices/2022/Bennett.md]: The readme mentioned that we could delete the backticks! I vote that you can get rid of all of those throughout this doc.

File: about-the-apprentices/2022/Bennett.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+One of my favorite things about programming is `critical thinking and the satisfaction of completing a difficult problem`. One thing I'm excited about learning is `test driven development`.
+
+## Icebreakers
+
+* What technology innovation made the most impact on your life
+`Standing desks`
```
Comment (marissahuysentruyt) [inline about-the-apprentices/2022/Bennett.md]: 💯 I just figured out how to readjust mine, and I was SO proud

**Review by marissahuysentruyt — CHANGES_REQUESTED** [review]
My only changes would be to get rid of the backticks!

File: about-the-apprentices/2022/Bennett.md (hunk 1 above)
Comment (bsmrdel101) [reply about-the-apprentices/2022/Bennett.md]: Ok made those changes.

File: .gitmessage [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# <type>(<scope>): <subject>
```
Comment (jordanleven) [inline .gitmessage]: Hey @bsmrdel101 - quick question: what's this file being used for here? Should this be tracked in the project or just your own personal gitconfig?

**Review by jordanleven — COMMENTED** [review]
Hey @bsmrdel101 - I just noticed something small on your PR you might want to check out.

**Review by marissahuysentruyt — APPROVED** [review]

File: .gitmessage (hunk 3 above)
Comment (bsmrdel101) [reply .gitmessage]: I was looking at the link posted in this repo, about Conventional Commits. [Here](https://github.com/sparkbox/standard/tree/main/code-style/git#the-art-of-the-commit-message)

Comment (jordanleven) [reply .gitmessage]: Hey @bsmrdel101  - yeah, that's the correct format, but this is something that you'd set in your local personal gitcofig settings and not in the project itself. You could store this file anywhere on your machine, but folks usually [put in in their home directory](https://dev.to/vetswhocode/git-gud-create-a-gitmessage-4ibj).

Comment (bsmrdel101) [reply .gitmessage]: Ok I moved the file.

**Comment by jordanleven** [comment]
This looks great! @bsmrdel101  - feel free to squash this into a single command and push up. After that, @marissahuysentruyt can pull down the squashed branch by using `git pull --rebase` and then merge it into `main`.

**Pull request merged**